_concepts/agency-summary.md

```diff
@@ -11,32 +11,50 @@ category: "Summary Pages:"
 headline: Agency Summary
 
 copy:
-  - text: Below is an idea on how to display data for a specific federal agency. Please take a look and give us your feedback in the discussion section at the bottom of each tab.
+  - text: Below are two idea on how to display data by federal agency. Please take a look and give us your feedback in the discussion section at the bottom of each tab.
 
 discussion_title: "Discussion suggestions:"
 
 discussion_bullet_points:
-- Does the layout and information help you complete your tasks?
-- Is there anything that should be removed from the layout?
+- Would you like additional information or data? 
+- Would you like the data displayed in a different way?
 - Are the labels and terminology appropriate?
-- Are the groupings of data sensible?
-- What do you like and dislike?
+- Are the filters helpful?
 
 tabs_required: yes
 
 tabs:
-  - title: Version 0.1
-    url_hash: "#!version-0.1"
+  - title: Version 0.1A
+    url_hash: "#!version-0.1A"
     archived_text:
     description: "Key Features:"
     features:
-      - lorum ipsum
-    disclaimer_text: <a class="usa-cta external-link" hr' target="_blank">View an interactive version of the below image</a>
-    image: " "
-    image_alt_text: This image displays 
+      - Agency's mission
+      - Breakdown of spending from budget to programs and recipients
+      - Spending by geography
+      - Table of award transactions by fiscal year
+    disclaimer_text: <a class="usa-cta external-link" href='http://kuv6vy.axshare.com/agency_summary_-_a.html' target="_blank">View an interactive version of the below image</a>
+    image: "/assets/img/concept_agency-summary_A.jpg.jpg"
+    image_alt_text: This image displays information about the objectives of the agency's mission across the top, along with the url to the agency website, the number of employees, the Washington D.C. address, and contact information. At the far right is the name and contact information for the Senior Agency Office. Below is a Spending Overview for Fiscal Year 2015, including the total budget amount, total spend amount, and spending broken down by Operational, Contracts, Grants, Loans, and Financial Assistance cagegories. In the next row are a horizontal bar chart showing highest funded programs and a map displaying spending by geography. In the next row are a horizontal bar chart displaying the highest awarded recipients and a pie chart showing spending by object class. In the next row is a timeline of agency events, and across the bottom of the page is a table displaying the agency awards by spending type.  To the far left are filters related to the categories. 
     below_image_content:
     disqus_identifier: 33785410
```

Comment:
This needs to be different than the other `disqus_identifier`. They have to be unique across all pages, since they're how disqus knows which comments to load.

To get a unique ID, you can just type a random number, or use something like http://www.currenttimestamp.com/ (which is a number representing the current time), so whenever you visit the site it'll be a new number.